.github/workflows/zfs-tests-functional.yml

```diff
@@ -54,15 +54,19 @@ jobs:
     - name: Clear the kernel ring buffer
       run: |
         sudo dmesg -c >/var/tmp/dmesg-prerun
+    - name: Report CPU information
+      run: |
+        lscpu
+        cat /proc/spl/kstat/zfs/chksum_bench
     - name: Report disk space
       run: |
         df -h /
     - name: Tests
       run: |
         set -o pipefail
-        /usr/share/zfs/zfs-tests.sh -vKR -s 3G | scripts/zfs-tests-color.sh
+        /usr/share/zfs/zfs-tests.sh -vKR -s 2G | scripts/zfs-tests-color.sh
```

Comment:
This change is responsible for some of the new test failures.  We've got a handful of tests which expect a slightly larger minimum pool size.

Reply:
okay, I will revert this :(